test/units/staking/stakeManager/StakeManager.test.js

```diff
@@ -342,6 +343,85 @@ contract('StakeManager', async function (accounts) {
         testRevert()
       })
     })
+
+    describe('checkSignatures with votes', function () {
```

Comment:
just "with votes" it is already part of checkSignatures

Reply:
done